qiita_pet/handlers/analysis_handlers.py

```diff
@@ -75,7 +76,7 @@ def post(self):
         name = self.get_argument('name')
         desc = self.get_argument('description')
         analysis = Analysis.create(self.current_user, name, desc,
-                                   from_default=True)
+                                   qiita_config.portal, from_default=True)
```

Comment:
can make it part of the config object. That way I can also make sure the portal passed actually exists before starting the web server.